helix-term/src/ui/picker.rs

```diff
@@ -276,30 +285,6 @@ pub struct Picker<T: 'static + Send + Sync, D: 'static> {
 }
 
 impl<T: 'static + Send + Sync, D: 'static + Send + Sync> Picker<T, D> {
-    pub fn stream(
```

Comment:
I'm not sure we will end up using these `stream`+`with_stream` helpers again (I believe #9647 refactored out the only caller) but I think it's premature to remove them now and it's an unrelated change, so let's reset these deletions

Reply:
I reverted these deletions.